gradoop-flink/src/main/java/org/gradoop/flink/model/impl/operators/aggregation/functions/max/MaxProperty.java

```diff
@@ -37,6 +37,6 @@ public MaxProperty(String propertyKey) {
 
   @Override
   public String getAggregatePropertyKey() {
-    return "max(" + propertyKey + ")";
+    return "max_" + propertyKey;
```

Comment:
Loading the `FlinkAsciiGraphLoader` from string is not working with the brackets, so I changed  this to be able to use the already existing aggregators.